torch/csrc/jit/tensorexpr/codegen_external.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import argparse
+from tools.codegen.gen import parse_native_yaml
+import tools.codegen.model as model
+from tools.codegen.code_template import CodeTemplate
+
+def deindent(code: str) -> str:
+    lines = code.split('\n')
+    min_leading_spaces = min(map(num_leading_spaces, lines))
+    lines = [line[min_leading_spaces:] for line in lines]
+    return '\n'.join(lines)
+
+
+def gen_external(native_functions_path):
+    native_functions = parse_native_yaml(native_functions_path)
+    func_decls = []
+    for schema in native_functions:
+        func = schema.func
+        name = func.name.name.base
+        args = func.arguments
+        # Only supports extern calls for functions with out variants
+        if func.name.overload_name != 'out':
```

Comment:
nit: you probably can use `func.is_out_fn` directly: https://github.com/pytorch/pytorch/blob/master/tools/codegen/model.py#L664